doc/developer_guide/transformations.rst

```diff
@@ -33,6 +33,22 @@
 .. -----------------------------------------------------------------------------
 .. Written by R. W. Ford, A. R. Porter and S. Siso, STFC Daresbury Lab
 
+.. testsetup::
+
+    import psyclone.psyir.nodes.node
+    def new_colored(text, _):
+        return text
+
+    # Disable colouring in output to allow passing of tests
```

Comment:
Please could you add a TODO for #1283 here as once that's done we can turn-off colouring when calling the `view()` method.

Reply:
I believe you meant #1238 (`node.view()` return string)?